### PR TITLE
Document TrustSDK specs

### PR DESCRIPTION
**Pull request by leoneparise**

### CODE REVIEW

File: trust-sdk/trust-sdk-specs.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+## URL Specifications
+
+TrustSDK works by calling deeplinks between apps, this document describes URL formats for each command. Due to the SDK high customization provided by TrustSDK, clients and wallets can adapt adapt the URL to it's own needs.
```
Comment (catenocrypt) [inline trust-sdk/trust-sdk-specs.md:3]: > clients and wallets can adapt adapt the URL to it's own needs

clients and wallets can adapt the URLs to their own needs

File: trust-sdk/trust-sdk-specs.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+## URL Specifications
+
+TrustSDK works by calling deeplinks between apps, this document describes URL formats for each command. Due to the SDK high customization provided by TrustSDK, clients and wallets can adapt the URLs to their own needs.
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:3]:
```suggestion
TrustSDK works by calling deeplinks between apps, this document describes URL formats for each command. TrustSDK is highly customizable, clients and wallets can adapt the URLs to their own needs. 
```

File: trust-sdk/trust-sdk-specs.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+### Sign
+
+Sign command is used to sign and broadcast transactions following [wallet-core](/wallet-core/wallet-core.md) protobuf models encoded in base64.
+
+#### Command
+* Command: `sdk_sign`
+* Parameters:
+  * `coin`: BIP44 coin code
+  * `data`: wallet-core protobuf model binary encoded in base64
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:27]:
```suggestion
  * `data`: wallet core protobuf model binary encoded in base64
```

File: trust-sdk/trust-sdk-specs.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+* `[scheme]` is the wallet's deeplink scheme. Default to `trust`
+* `[command]` is the command to be executed
+* `[params]` is the command parameters encoded as url query items.
+* `[id]` is the unique command id. The value is later used to resolve the callback for the command. Can be any incrementing integer.
+* `[app]` callback URL scheme. 
+* `[callback]`callback path. Default to `sgn_sign_result`
+
+### Sign
+
+Sign command is used to sign and broadcast transactions following [wallet-core](/wallet-core/wallet-core.md) protobuf models encoded in base64.
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:21]:
```suggestion
Sign command is used to sign and broadcast transactions leveraging [wallet core](/wallet-core/wallet-core.md) protobuf models (encoded in base64).
```

File: trust-sdk/trust-sdk-specs.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+#### Command
+* Command: `sdk_sign`
+* Parameters:
+  * `coin`: BIP44 coin code
+  * `data`: wallet-core protobuf model binary encoded in base64
+  * `send`: flag that indicates if the wallet should broadcast the transaction. Possible values: `true`, `false`.
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:28]:
```suggestion
  * `send`: flag that indicates if the wallet should broadcast the transaction. Valid values: `true`, `false`.
```

File: trust-sdk/trust-sdk-specs.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+  * `meta`: transaction metadata. This attribute is optional and encoded as a dictionary. Check how it's encoded in [Dictionary Encoding](#dictionary-encoding).
+    * `meta.__name`: transaction metadata type. Possible values: `dApp`
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:30]: dApp or dapp?
```suggestion
    * `meta.__name`: transaction metadata type. Possible values: `dapp`?
```

File: trust-sdk/trust-sdk-specs.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+lrzEHpAAIAmoB0Yegc2ZqxtkSkXlYo_TEg4eBrjopGUj9ySxJh6JlfToGqR7yNKzV8cD_yN_jVR5YrVaTANO05X2_
+9HleO8htQqEgEmGiAdGHoHNmasbZEpF5WKP0xIOHga46KRlI_cksSYeiZX0yIgapHvI0rNXxwP_I3-NVHlitVpMA0
+7Tlfb_0eV47yG1Co&id=4
+```
+
+### Get Accounts
+
+#### Command
+
+* Command: `sdk_sign`
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:62]: Should we put "Get Accounts" in front of "Sign"?
```suggestion
* Command: `sdk_get_accounts`
```

**Comment by hewigovens** [comment]
Should we add the ability to callback different url scheme? in `callback`, not always from the source app

File: trust-sdk/trust-sdk-specs.md (hunk 6 above)
Comment (leoneparise) [reply trust-sdk/trust-sdk-specs.md:30]: `dApp`

File: trust-sdk/trust-sdk-specs.md (hunk 7 above)
Comment (leoneparise) [reply trust-sdk/trust-sdk-specs.md:62]: Good catch

File: trust-sdk/trust-sdk-specs.md (hunk 6 above)
Comment (hewigovens) [reply trust-sdk/trust-sdk-specs.md:30]: the example request is still `dapp`

File: trust-sdk/trust-sdk-specs.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+trust://sdk_get_accounts?coins.0=60&coins.1=714&app=sampleapp&callback=sdk_sign_result&id=2
+```
+
+#### Response
+* Parameters:
+  * `accounts`: Public addresses joined by `,`.
+
+##### Example
+```
+sampleapp://sdk_sign_result?accounts=0xFC6CD054fAc48Df3744B42ea82E1Dd0fa7027086,
```
Comment (hewigovens) [inline trust-sdk/trust-sdk-specs.md:77]: typo sdk_sign_result ?

File: trust-sdk/trust-sdk-specs.md (hunk 6 above)
Comment (catenocrypt) [reply trust-sdk/trust-sdk-specs.md:30]: case-insensitive check is a good idea in such cases (just to consider)

**Pull request merged**